### PR TITLE
fix: supports cross platform builds

### PR DESCRIPTION
**Pull request by brennanMKE**
*Issue #, if available:*

#97

*Description of changes:*

Adds platform versions to Package.swift.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by harsh62 — APPROVED** [review]

**Review by lawmicha — APPROVED** [review]

File: Package.swift [hunk 1]
```diff
@@ -1,10 +1,14 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.5
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "AppSyncRealTimeClient",
+    platforms: [
+        .iOS(.v13),
```
Comment (lawmicha) [inline Package.swift:9]: wait i think... iOS AppSync SDK still uses this, will that impact the SDK to take on new versions of this package?

**Review by lawmicha — CHANGES_REQUESTED** [review]
need to verify iOS AppSync SDK dependency on this package, and for within this package, we also have pod support (does PodSpec need to be updated?)

File: Package.swift (hunk 1 above)
Comment (lawmicha) [reply Package.swift:9]: https://github.com/awslabs/aws-mobile-appsync-sdk-ios

Comment (brennanMKE) [reply Package.swift:9]: Yes, this dependency has to match the amplify-ios platform requirements to be compatible.

**Review by lawmicha — APPROVED** [review]

**Pull request merged**